lib/philomena_web/controllers/api/json/posts_controller.ex

```diff
@@ -44,6 +66,14 @@ defmodule PhilomenaWeb.Api.Json.PostController do
       |> preload([_p, t, _f], topic: t)
       |> Repo.all()
 
-    json(conn, %{posts: Enum.map(posts, &PostJson.as_json/1), page: page, total: hd(posts).topic.post_count})
+    case posts do
```

Comment:
Is this the appropriate response? Suppose you page to the middle of a topic that has 25 destroyed posts in a row. This would end up being 404 ...